### PR TITLE
MarketplaceV2: Add list of supported MPs to yml files

### PR DESCRIPTION
**Pull request by Bargenish**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
related: https://github.com/demisto/etc/issues/44708

## Description
Phase 1:
Added the marketplace field to all yml files.
Excluded:
* unsupported types
* deprecated content

### CODE REVIEW

File: Tests/Marketplace/Tests/edit_pack_metadata.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
+import argparse
```
Comment (yaakovi) [inline Tests/Marketplace/Tests/edit_pack_metadata.py]: move to `Tests/Marketplace/Utils/edit_pack_metadata.py`

File: Tests/Marketplace/Tests/edit_pack_metadata.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+def main():
+    option = option_handler()
+    key_to_add = option.key_name
+    value = option.value
+
+    if provided_packs := option.packs:
+        packs = [os.path.join(PACKS_DIR, p) if PACKS_DIR not in p else p for p in provided_packs.split(',')]
+    else:
+        packs = list(filter(os.path.isdir, [os.path.join(PACKS_DIR, p) for p in os.listdir(PACKS_DIR)]))
```
Comment (yaakovi) [inline Tests/Marketplace/Tests/edit_pack_metadata.py]:
```suggestion
        pack_names = next(os.walk(PACKS_DIR))[1]
        packs = [os.path.join(PACKS_DIR, p) for p in pack_names]
```

**Review by yaakovi — COMMENTED** [review]
need to add the `marketplaces` field to the item schema before merging this PR, otherwise, it will break master.

**Comment by Bargenish** [comment]
@yaakovi the script wont be added to this PR I'll remove it.
Will open the changes on the schemas today .

**Pull request closed without merging**